docarray/typing/abstract_type.py

```diff
@@ -13,6 +13,8 @@
 
 
 class AbstractType(BaseNode):
+    is_tensor = False  # change for tensor-like subclasses
```

Comment:
I am not sure to understand why we need this. Only object of type AbstractTensor will have this `is_tensor=True` right ? So why not just do
```python
if issubclass(type_, AbstractType):
    if issubclass(type_, AbstractTensor):
        smth_with_tensor()
    else:
        smth_without_tensor

```

Reply:
Yeah true, will revert